README.md

```diff
@@ -1 +1,72 @@
-# ds-helper
\ No newline at end of file
+---
+__Let's Learn Some Basic GitHub Steps :)__
+
+- __[KOMEN.CO](http://www.estudiosdemercado.org/komenco.jpg)__ -The Beginning of your Journey
+![Minion](http://www.estudiosdemercado.org/komenco.jpg)
```

Comment:
Can you please use this image as cover?
![KOMEN CO](https://user-images.githubusercontent.com/16015962/109482637-3e194200-7aa4-11eb-920e-0dc7aa891e1b.png) and use it to link to https://www.komen.co.in

Reply:
Sir updated the image, kindly review % suggest further changes if required

Reply:
*if

Reply:
Sir Updated the image, kindly review & suggest if the further change is required

Reply:
Sir Updated the image, kindly review & suggest if further change is required